tests/all/integration/api/helpers/test_order.py

```diff
@@ -54,6 +56,120 @@ def test_should_delete_related_attendees(self):
+            third_att_with_completed_order.order_id = third_completed_order.id
+            fourth_att_with_completed_order = AttendeeFactoryBase()
+            fourth_att_with_completed_order.order_id = fourth_completed_order.id
+            fifth_att_with_completed_order = AttendeeFactoryBase()
+            fifth_att_with_completed_order.order_id = fifth_completed_order.id
+            sixth_att_with_completed_order = AttendeeFactoryBase()
+            sixth_att_with_completed_order.order_id = sixth_completed_order.id
+
+            att_with_placed_order = AttendeeFactoryBase()
+            att_with_placed_order.order_id = placed_order.id
+
+            first_att_with_init_order = AttendeeFactoryBase()
+            first_att_with_init_order.order_id = first_initialized_order.id
+            second_att_with_init_order = AttendeeFactoryBase()
+            second_att_with_init_order.order_id = second_initialized_order.id
+            third_att_with_init_order = AttendeeFactoryBase()
+            third_att_with_init_order.order_id = third_initialized_order.id
+            fourth_att_with_init_order = AttendeeFactoryBase()
+            fourth_att_with_init_order.order_id = fourth_initialized_order.id
+
+            first_att_with_expired_time = AttendeeFactoryBase()
+            first_att_with_expired_time.order_id = first_expired_time_order.id
+            second_att_with_expired_time = AttendeeFactoryBase()
+            second_att_with_expired_time.order_id = second_expired_time_order.id
+            third_att_with_expired_time = AttendeeFactoryBase()
+            third_att_with_expired_time.order_id = third_expired_time_order.id
+
+            first_att_with_expired_order = AttendeeFactoryBase()
+            first_att_with_expired_order.order_id = first_expired_order.id
+            second_att_with_expired_order = AttendeeFactoryBase()
+            second_att_with_expired_order.order_id = second_expired_order.id
+            third_att_with_expired_order = AttendeeFactoryBase()
+            third_att_with_expired_order.order_id = third_expired_order.id
+            fourth_att_with_expired_order = AttendeeFactoryBase()
+            fourth_att_with_expired_order.order_id = fourth_expired_order.id
+            fifth_att_with_expired_order = AttendeeFactoryBase()
+            fifth_att_with_expired_order.order_id = fifth_expired_order.id
+
+            db.session.commit()
+
+            order_expiry_time = get_settings()['order_expiry_time']
+            count = db.session.query(Order.id) \
+                .filter(Order.event_id == int(ticket.event_id),
+                        Order.deleted_at.is_(None),
+                        or_(Order.status == 'placed', Order.status == 'completed',
+                            and_(Order.status == 'initializing',
+                                 Order.created_at + timedelta(minutes=order_expiry_time) >
+                                 datetime.utcnow()))).count()
+
+            self.assertEqual(count, 11)
```

Comment:
@iamareebjamal - I tested the count which query returnd here.

Reply:
Yeah, exactly. This test tests itself. What's the use of that? We need a test which tests the query written in the app, not the test itself? What good is testing 2 + 2 == 4?